plugins/item_tasks/web_client/views/ControlWidget.js

```diff
@@ -191,14 +194,38 @@ var ControlWidget = View.extend({
             help = 'Browse to a directory to select it, then click "Save"';
         }
 
+        if (type === 'new-folder') {
+            title = 'Create a new folder';
+            help = 'Browse to a path, enter a name, then click "Save"';
+            input = {
+                label: 'Name',
+                placeholder: 'Choose a name for the new folder',
+                validate: (val) => {
+                    // validation on the "new item name"
+                    if (!val) {
+                        return 'Please provide an folder name.';
+                    }
+                },
+                default: this.model.get('fileName')
```

Comment:
I'm not sure. Is this task spec outlined anywhere? I tried looking at girder_worker but didn't find comprehensive format specifications.